apps/openmw/mwbase/world.hpp

```diff
@@ -438,12 +438,16 @@ namespace MWBase
 
             virtual void enableActorCollision(const MWWorld::Ptr& actor, bool enable) = 0;
 
-            virtual int canRest() = 0;
-            ///< check if the player is allowed to rest \n
-            /// 0 - yes \n
-            /// 1 - only waiting \n
-            /// 2 - player is underwater \n
-            /// 3 - enemies are nearby (not implemented)
+            enum RestPermitted
+            {
+                Rest_Allowed = 0,
+                Rest_OnlyWaiting = 1,
+                Rest_PlayerIsUnderwater = 2,
+                Rest_EnemiesAreNearby = 3
+            };
+
+            virtual RestPermitted canRest() const = 0;
+            ///< check if the player is allowed to rest
```

Comment:
We don't use this style of comment (///<, comment after function) anymore, because is causes problems with some editors/IDE. Now we don't like people doing commits with purely stylistic changes either (because that increases the chance of merge conflicts), but since you are touching this line anyway, I suggest to fix it (remove the < and then move the comment to the top).